x/staking/types/keys.go

```diff
@@ -19,6 +19,9 @@ const (
 	// StoreKey is the string store representation
 	StoreKey = ModuleName
 
+	// TStoreKey is the string transient store representation
+	TStoreKey = ModuleName
```

Comment:
What is a transient store? Does it only exist for a block?

Reply:
IIRC, a transient store is a store that is not persisted to disk and whose state exists block by block. Correct me if I'm wrong @cwgoes

Reply:
yeah, that's right